docs/release_notes.rst

```diff
@@ -22,6 +22,9 @@ New Features
   will now be checked by default instead of D203, which required a single
   blank line before a class docstring (#137).
 
+* Now handles configuration filed correctly. The closer a configuration file
```

Comment:
Phrase this as "Configuration files are now handled..." etc. Also filed -> files.